### ISSUE TITLE
Search results issues

### ISSUE THREAD

**Issue by trongthanh** [naustudio/nau-jukebox#35]
- Lack keyboard navigation after React convert
- When the result panel loose focus, its background is almost transparent and its text overlapping with below texts (reduce transparency when loose focus)
